zrml/neo-swaps/src/weights.rs

```diff
@@ -57,187 +57,281 @@ pub trait WeightInfoZeitgeist {
+    /// Storage: `Tokens::TotalIssuance` (r:128 w:128)
+    /// Proof: `Tokens::TotalIssuance` (`max_values`: None, `max_size`: Some(57), added: 2532, mode: `MaxEncodedLen`)
+    /// The range of component `n` is `[1, 7]`.
+    fn combo_sell(n: u32) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `0 + n * (3627 ±0)`
+        //  Estimated: `156294 + n * (38153 ±484)`
+        // Minimum execution time: 523_942 nanoseconds.
+        Weight::from_parts(527_472_000, 156294)
+            // Standard Error: 38_920_367
+            .saturating_add(Weight::from_parts(1_535_695_671, 0).saturating_mul(n.into()))
+            .saturating_add(T::DbWeight::get().reads(10))
+            .saturating_add(T::DbWeight::get().reads((23_u64).saturating_mul(n.into())))
+            .saturating_add(T::DbWeight::get().writes(9))
+            .saturating_add(T::DbWeight::get().writes((22_u64).saturating_mul(n.into())))
+            .saturating_add(Weight::from_parts(0, 38153).saturating_mul(n.into()))
+    }
+    /// Storage: `MarketCommons::Markets` (r:7 w:0)
+    /// Proof: `MarketCommons::Markets` (`max_values`: None, `max_size`: Some(708), added: 3183, mode: `MaxEncodedLen`)
+    /// Storage: `System::Account` (r:2 w:2)
+    /// Proof: `System::Account` (`max_values`: None, `max_size`: Some(132), added: 2607, mode: `MaxEncodedLen`)
+    /// Storage: `Tokens::Accounts` (r:382 w:382)
+    /// Proof: `Tokens::Accounts` (`max_values`: None, `max_size`: Some(137), added: 2612, mode: `MaxEncodedLen`)
+    /// Storage: `Tokens::TotalIssuance` (r:254 w:254)
+    /// Proof: `Tokens::TotalIssuance` (`max_values`: None, `max_size`: Some(57), added: 2532, mode: `MaxEncodedLen`)
+    /// Storage: `NeoSwaps::PoolCount` (r:1 w:1)
+    /// Proof: `NeoSwaps::PoolCount` (`max_values`: Some(1), `max_size`: Some(16), added: 511, mode: `MaxEncodedLen`)
+    /// Storage: `NeoSwaps::Pools` (r:0 w:1)
+    /// Proof: `NeoSwaps::Pools` (`max_values`: None, `max_size`: Some(152829), added: 155304, mode: `MaxEncodedLen`)
+    /// The range of component `n` is `[1, 7]`.
+    fn deploy_combinatorial_pool(n: u32) -> Weight {
+        // Proof Size summary in bytes:
+        //  Measured:  `357 + n * (185 ±0)`
+        //  Estimated: `11438 + n * (57229 ±969)`
+        // Minimum execution time: 7_103_129 nanoseconds.
+        Weight::from_parts(7_159_730_000, 11438)
+            // Standard Error: 1_623_272_081
+            .saturating_add(Weight::from_parts(61_965_055_407, 0).saturating_mul(n.into()))
+            .saturating_add(T::DbWeight::get().reads(10))
+            .saturating_add(T::DbWeight::get().reads((37_u64).saturating_mul(n.into())))
+            .saturating_add(T::DbWeight::get().writes(10))
+            .saturating_add(T::DbWeight::get().writes((37_u64).saturating_mul(n.into())))
+            .saturating_add(Weight::from_parts(0, 57229).saturating_mul(n.into()))
+    }
```

Comment:
This is pretty costly. The maximum block ref weight is `500_000_000_000` currently. This would allow not even eight markets.